pkg/scheduler/framework/plugins/nodeunschedulable/node_unschedulable_test.go

```diff
@@ -170,6 +170,17 @@ func TestIsSchedulableAfterNodeChange(t *testing.T) {
 			},
 			expectedHint: framework.Queue,
 		},
+		{
+			name:   "skip-original-schedulable-node-modified-to-nil",
```

Comment:
It shouldn't happen, right? newObj == nil is the deletion event, which this plugin doesn't subscribe.

Reply:
yes, we only have Add and Update event